README.md

```diff
@@ -5,12 +5,21 @@
 
 
 ### Data going from the client to the server can get lost
-  
-* Explain failure modes/effects
-* Call failed but data isn't lost
-* Call failed and data is lost
-* Ergo, a non-success http response means that you don't know what happened to your data
-* Even not losing data might be an issue since the client doesn't know that it was stored?
+
+Getting data from the browser to the web-server is exposed to the fallacies of distributed computing. Lots have been written on that topic but in short:
+
+>things can go wrong, horribly wrong 
+
+The request might never reach the server with a http timeout as a result, it might reach the server but something bad might happen before your data is safely stored with a http 500 Internal Server error as a result. It doesn't end there, things might go wrong after data is stored, same 500 will greet the client. Are we done? Nope, something might go wrong on the way back to the client, http timeout again. As you can see lots of things can happen but from a client perspective we can summarize it pretty simple as: 
+
+>any non success status code means that you have no idea what happened to your data, it might be safe or it might not 
```

Comment:
I think we should be more specific: 4xx are fine as the client should assume that the request has not been processed.

Reply:
Makes sense, clarified this